daheinze-oscheer/hello/hello.go

```diff
@@ -0,0 +1,33 @@
+package main
```

Comment:
Package should have a meaningful name and description, like **package hello**

```suggestion
// Package hello contains code to send a nice greeting in some languages
package hello
```